package.json

```diff
@@ -72,14 +72,14 @@
     "nuxt-build-optimisations": "latest",
     "nuxt-typed-vuex": "latest",
     "oh-vue-icons": "^0.4.7",
+    "rif-aggregation": "^0.11.8",
```

Comment:
I know this is annoying, but if we change the name as [suggested](https://github.com/rsksmart/ri-aggregation/pull/28), we need to update it here.

Reply:
Updated